.vscodeignore

```diff
@@ -1,17 +1,14 @@
-# Ignore everything
-*
-*/**
-
-# Whitelist required files
-!dist/**/*
-!images/**/*
-!snippets/**/*
-!syntaxes/**/*
-
-!.vscodeignore
-!CHANGELOG.md
-!ecl.configuration.json
-!kel.configuration.json
-!LICENSE
-!package.json
-!README.md
+.github
+.vscode
+lib-es6
+node_modules
+samples
+src
+.gitignore
+.vscodeignore
+ojs.configuration.json
+package-lock.json
+test.html
+tsconfig.json
+tslint.json
+webpack.config.js
```

Comment:
Thanks for pointing me in the correct direction (resolved in master now).

FWIW this was closer to my previous attempt:

```
# Ignore everything
*

# Whitelist required files
!dist/
!dist/*
!images/
!images/*
!syntaxes/
!syntaxes/*

!.vscodeignore
!CHANGELOG.md
!ojs.configuration.json
!LICENSE
!package.json
!README.md
```